NitroxServer/Communication/NetworkingLayer/NitroxServer.cs

```diff
@@ -74,6 +74,15 @@ protected void SetupUPNP()
             NatUtility.StartDiscovery();
         }
 
+        public PlayerManager GetPlayerManager()
+        {
+            return playerManager;
+        }
+        public int GetPlayerCount()
+        {
+            return playerManager.GetConnectedPlayers().Count;
+        }
+
```

Comment:
We use Autofac to get these objects.
Eg. `NitroxServiceLocator.LocateService<PlayerManager>();` or with parameters and initializing it with Autofac (see [here](https://github.com/SubnauticaNitrox/Nitrox/blob/c7dc535fc8658edb220363ba68fb5fab2675a3d6/NitroxServer/ServerAutoFacRegistrar.cs#L33)).

So pls remove any unnecessary getter/setter and use one of the options above.